coordinator/internal/orm/batch.go

```diff
@@ -83,7 +83,7 @@ func (*Batch) TableName() string {
 func (o *Batch) GetUnassignedBatch(ctx context.Context, maxActiveAttempts, maxTotalAttempts uint8) (*Batch, error) {
 	var batch Batch
 	db := o.db.WithContext(ctx)
-	sql := fmt.Sprintf("SELECT * FROM batch WHERE proving_status = %d AND total_attempts < %d AND active_attempts < %d AND chunk_proofs_status = %d AND batch.deleted_at IS NULL ORDER BY batch.index LIMIT 1;",
+	sql := fmt.Sprintf("SELECT * FROM batch WHERE proving_status = %d AND total_attempts < %d AND active_attempts < %d AND chunk_proofs_status = %d AND codec_version != 5 AND batch.deleted_at IS NULL ORDER BY batch.index LIMIT 1;",
```

Comment:
Why add `codec_version != 5 AND`? coordinator don't need to care about this, it believes rollup_relayer's check, because they use the same db/table.

Reply:
`codec_version = 5`is the special euclid transition batch which doesn't need any proving. So coordinator should just ignore it.